data/morphology/unimorph_languages.json

```diff
@@ -1,4 +1,20 @@
 {
-    "eng": "http://raw.githubusercontent.com/unimorph/eng/master/eng",
-    "ger": "http://raw.githubusercontent.com/unimorph/deu/master/deu"
-}
+        "ady": "https://raw.githubusercontent.com/unimorph/ady/master/ady",
+        "arm": "https://raw.githubusercontent.com/unimorph/hye/master/hye",
+        "aze": "https://raw.githubusercontent.com/unimorph/aze/master/aze",
+        "bak": "https://raw.githubusercontent.com/unimorph/bak/master/bak",
+        "bel": "https://raw.githubusercontent.com/unimorph/bel/master/bel",
+        "bul": "https://raw.githubusercontent.com/unimorph/bul/master/bul",
+        "cat": "https://raw.githubusercontent.com/unimorph/cat/master/cat",
+        "ces": "https://raw.githubusercontent.com/unimorph/ces/master/ces",
+        "dan": "https://raw.githubusercontent.com/unimorph/dan/master/dan",
+        "dut": "https://raw.githubusercontent.com/unimorph/nld/master/nld",
+        "eng": "https://raw.githubusercontent.com/unimorph/eng/master/eng",
+        "fao": "https://raw.githubusercontent.com/unimorph/fao/master/fao",
+        "fin_1": "https://raw.githubusercontent.com/unimorph/fin/master/fin.1",
```

Comment:
This is a TODO for a later PR but we need a way to concatenate these after the fact.

What I'd suggest is that instead of just having a single URL, we have list of URLs, and then all URLs are written to `fin.tsv`. Not a big change.

Reply:
ok but this should get filed separately, correct?

Reply:
What you did here is definitely fine for now.

Reply:
I couldn't find unimorph files in the [repo](https://unimorph.github.io/) list with repos for afar (aar), asamese (asm)or esperanto (epo). Is there somewhere else I should look or did they not get updated in the last iteration?

Reply:
That's fine. But there are no language beyond "g" at all right now.